packages/model-viewer/src/test/model-viewer-base-spec.ts

```diff
@@ -99,6 +99,28 @@ suite('ModelViewerElementBase', () => {
       });
     });
 
+    suite('with modelViewer global config', () => {
+      ModelViewerElementBase.powerPreference = 'low-power'
+      let element: ModelViewerElementBase;
+
+      setup(() => {
+        element = new ModelViewerElement();
+        document.body.insertBefore(element, document.body.firstChild);
+      });
+
+      teardown(() => {
+        element.parentNode!.removeChild(element);
+      });
+
+      test(
+          'validate threeRenderer is using the given powerPreference',
+          async () => {
+            const {threeRenderer} = Renderer.singleton;
+            expect(threeRenderer.getContextAttributes().powerPreference)
+                .to.be.equal('low-power');
+          });
+    });
```

Comment:
Another side effect is that this will no longer be auto-testable. This also describes a common issue we have with testing in the browser, which is that a true end-to-end test that really checks what you care about can be pretty impossible in some situations.

Reply:
Would that mean I should revert this change?

Reply:
Yes